.ansible-lint

```diff
@@ -1,3 +1,4 @@
 ---
 skip_list:
 - '602'
+- '106'
```

Comment:
I assume it is not possible to fix the error about the role name, right?

Reply:
right